core/src/main/java/org/axonframework/eventsourcing/eventstore/EventStore.java

```diff
@@ -77,4 +79,19 @@ default DomainEventStream readEvents(String aggregateIdentifier, long firstSeque
      * @param snapshot The snapshot to replace part of the DomainEventStream.
      */
     void storeSnapshot(DomainEventMessage<?> snapshot);
+
+    /**
+     * Returns the last known sequence number of an Event for the given {@code aggregateIdentifier}.
+     * <p>
+     * It is preferred to retrieve the last known sequence number from the Domain Event Stream when sourcing an
+     * Aggregate from events. However, this method provides an alternative in cases no events have been read. For
+     * example when using state storage.
+     *
+     * @param aggregateIdentifier The identifier of the aggregate to find the highest sequence for
```

Comment:
Nit: Shouldn't there be dots at the end of the `@param` and `@return` descriptions?

Reply:
dito...